assets/js-examples-new/page-objects/google/searchResults.js

```diff
@@ -33,15 +33,16 @@ module.exports = {
 
   sections: {
     menu: {
-      selector: '#hdtb-msb',
+      selector: '//*[@id="main"]//div[@role="navigation"]',
+      locateStrategy: 'xpath',
```

Comment:
No need to mention `//*[@id="main"]` here since all the elements are anyway present inside the `main` element only (Adding the XPath for `main` does not help filter anything).

Also, the selector can be converted back "css selector" by using `'div[role="navigation"]'`. And you can filter it out further to get the specific element by using `'div[role="navigation"] div[data-st-cnt="mode"]'` as selector.